src/domains/gps-app/api/gpd_api/debt-position-services/v2/_base_policy.xml

```diff
@@ -1,5 +1,12 @@
 <policies>
   <inbound>
+    <choose>
+      <when condition="@(Regex.IsMatch(context.Request.Url.Path, @"^\/pagopa-gpd-core\/organizations\/([a-zA-Z0-9]{11})\/debtpositions$", RegexOptions.IgnoreCase, TimeSpan.FromMilliseconds(500)) && ((string)(context.Request.Method)).Equals("POST"))">
```

Comment:
as said before.
perhaps you can create a fragment too, where the input params are path and serviceType values